runtime/oti/j9.h

```diff
@@ -357,7 +357,7 @@ static const struct { \
 	((NULL != (method)) && (J9ROMCLASS_IS_ANON_OR_HIDDEN(J9_CLASS_FROM_METHOD((method))->romClass) || J9_ARE_ANY_BITS_SET(J9_ROM_METHOD_FROM_RAM_METHOD((method))->modifiers, J9AccMethodFrameIteratorSkip)))
 
 #define IS_VIRTUAL_THREAD(vmThread, object) \
-	isSameOrSuperClassOf(J9VMJAVALANGBASEVIRTUALTHREAD_OR_NULL(((vmThread)->javaVM)), J9OBJECT_CLAZZ((vmThread), (object)))
+	isSameOrSuperClassOf(J9VMJAVALANGBASEVIRTUALTHREAD(((vmThread)->javaVM)), J9OBJECT_CLAZZ((vmThread), (object)))
```

Comment:
While you're here, please remove the extraneous parentheses around the argument to `J9VMJAVALANGBASEVIRTUALTHREAD`. The parentheses around the arguments to `J9OBJECT_CLAZZ` are also unnecessary.

Reply:
Updated.